NektarDriftwave/README.md

```diff
@@ -0,0 +1,175 @@
+# NetkarDriftwave state space model
+
+Example integration of ParticleDA with a [Nektar++](https://www.nektar.info/) solver for a plasma physics model, produced as part of the 
+[ExCALIBUR fusion use case project NEPTUNE](https://excalibur.ac.uk/projects/excalibur-fusion-use-case-project-neptune-neutrals-plasma-turbulence-numerics-for-the-exascale/).
+
+This example combines the [`nektar-driftwave` solver](https://github.com/ExCALIBUR-NEPTUNE/nektar-driftwave) with simulation of Gaussian random fields with Matern covariance functions using Nektar++ to form a state space model suitable for applying a particle filter to. The deterministic dynamics solve the [two-dimensional Hasegawa-Wakatani equations](https://github.com/ExCALIBUR-NEPTUNE/nektar-driftwave/blob/master/doc.ipynb) with spatially correlated additive state noise simulated by solving a Helmholtz equation driven by a Gaussian white noise process. 
+
+The system is simulated on a rectangular spatial domain with periodic boundary conditions, and a regular quadrilateral mesh, using the Nektar++ spectral element method implementation. The Nektar++ mesh file is generated using [the Gmsh Julia interface](https://github.com/JuliaFEM/Gmsh.jl) and the [Nektar++ NekMesh utility](https://doc.nektar.info/userguide/5.4.0/user-guidech5.html), allowing the mesh dimensions and spatial size to be specified programmatically.
+
+## Prerequisites
+
+The example requires the following to be installed:
+
+- Nektar++ with HDF5 support (tested with [version 5.4](https://gitlab.nektar.info/nektar/nektar/-/tree/v5.4.0)) - [installation instructions](https://www.nektar.info/getting-started/installation/).
+- `nektar-driftwave` solver (tested with [commit `5a6e941`](https://github.com/ExCALIBUR-NEPTUNE/nektar-driftwave/tree/5a6e94131926116590a988058b651e653ac423b7)) - [installation instructions](https://github.com/ExCALIBUR-NEPTUNE/nektar-driftwave?tab=readme-ov-file#compilation--installation).
+- Julia (tested with version 1.9) - [installation instructions](https://julialang.org/downloads/).
+
+## Installation
+
+To set up a Julia environment with all the direct Julia dependencies for the example installed, from the `NektarDriftwave` directory in a local clone of this repository run
```

Comment:
Refactor as a package to allow installing using package manager by passing relevant GitHub URL using form
```
Pkg.add(url="https://github.com/Company/MonoRepo", subdir="juliapkgs/Package.jl)")
```